client/src/components/admin/AdminRound/AdminRoundContent.jsx

```diff
@@ -71,7 +86,9 @@ function AdminRoundContent({ round, competitionId, officialWorldRecords }) {
   const [batchResults, setBatchResults] = useState(() =>
     getStoreBatchResults(round.id),
   );
-  const [isBatchMode, setIsBatchMode] = useState(batchResults.length > 0);
+  const [isBatchMode, setIsBatchMode] = useState(
+    batchResults.length > 0 || getStoreIsBatchMode(),
```

Comment:
We can pass an initialiser function to `useState`. The advantage is that it runs only once to get the default value, and in this case it's preferable, since we don't want to read from local storage on every render.

```suggestion
    () => batchResults.length > 0 || getStoreIsBatchMode(),
```